src/components/settings-modal/settings-modal.js

```diff
@@ -62,6 +64,28 @@ const SettingsModal = ({
               }
             />
           ))}
+          {isOutdated ? (
+            <div className="pipeline-settings-modal__upgrade-reminder">
+              <span>&#8226;</span>
+              <span> Kedro-Viz {latestVersion} is here! </span>
+              <a
+                href="https://github.com/kedro-org/kedro-viz/releases"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                View release notes
+              </a>
+            </div>
+          ) : (
+            <div className="pipeline-settings-modal__already-latest">
+              <span>&#8226;</span>
+              <span>
+                {' '}
+                You are already on the latest version of Kedro-Viz (
```

Comment:
I love this. Thanks for adding!

Don't think `already` needs to be in this line based on the design:

![image](https://user-images.githubusercontent.com/16869061/154558160-7615f6eb-87ce-4fa4-bcdc-86c286071587.png)

Reply:
indeed - I've updated the wording.